docs/example.md

```diff
@@ -0,0 +1,51 @@
+# Example walkthrough of PFB
+![Example data PFB schema](diagrams/pfbExample.png)
+
+With some example row data that looks like the following:
+
+```
+{"id": "HG01101_cram", "name": "submitted_aligned_reads", "object": {"file_format": "BAM", "error_type": "file_size", "file_name": "foo.bam", "file_size": 512, "file_state": "registered", "md5sum": "bdf121aadba028d57808101cb4455fa7", "object_id": "dg.4503/cc32d93d-a73c-4d2c-a061-26c0410e74fa", "created_datetime": null, "ga4gh_drs_uri": "drs://example.org/dg.4503/cc32d93d-a73c-4d2c-a061-26c0410e74fa", "participant_id": "bbb1234", "specimen_id": "spec1111", "experimental_strategy": null, "study_registration": "example.com/study_registration", "study_id": "aaa1234", "project_id": "DEV-test", "state": "uploading", "submitter_id": "HG01101_cram", "subject_id": "p1011554-9", "updated_datetime": null}, "relations": []}
```

Comment:
Optional style update: put this in a json formatter so it's laid out in a more readable shape instead of in one big line.  I think you can also do "```json" at the beginning to get slightly improved syntax highlighting.